ctrl/ctrl1.go

```diff
@@ -531,6 +532,25 @@ func home(w http.ResponseWriter, r *http.Request) {
 		}
 		conn.Close()
 
+	case "rom_sol_log":
+		// We must get the username from the request
+		base.Zlog.Infof("ROM sol log start received")
+		// Truncate the file, if already exists
+		if _, fileerr := os.Stat(solLogPath + "bios_sol.log"); fileerr == nil {
+			os.Truncate(solLogPath+"bios_sol.log", 0)
+		}
+
+		// we need to start also the console
+		sutIP := r.FormValue("bmcip")
+		base.Zlog.Infof("SUTIP:", sutIP)
+
+		var argsConsole []string
+		argsConsole = append(argsConsole, sutIP)
+		romSerialConsoleCmd = exec.Command("/usr/local/production/bin/bioslog", argsConsole...)
```

Comment:
Could the path be an input parameter of the microservices instead of an hard coded value ?

Reply:
taking discussion offline ;-)